backend/typescript/prisma/schema.prisma

```diff
@@ -108,37 +108,37 @@ model role {
 }
 
 model resident {
-  id                   Int                 @id @default(autoincrement())
-  first_name           String
-  last_name            String
-  email                String              @unique
-  phone_number         String?
-  display_name         String
-  profile_picture_link String?
-  birthdate            DateTime?
-  credits              Float
-  date_joined          DateTime
-  date_left            DateTime?
-  tasks                task[]
-  warnings             warning[]
-  notifications        notification_user[]
+  id                 Int                 @id @default(autoincrement())
+  firstName          String              @map("first_name")
+  lastName           String              @map("last_name")
+  email              String              @unique
+  phoneNumber        String?             @map("phone_number")
+  displayName        String              @map("display_name")
+  profilePictureLink String?             @map("profile_picture_link")
+  birthdate          DateTime?
+  credits            Float
+  dateJoined         DateTime            @map("date_joined")
+  dateLeft           DateTime?           @map("date_left")
+  tasks              task[]
+  warnings           warning[]
+  notifications      notification_user[]
 }
 
 model notification {
-  id         Int                 @id @default(autoincrement())
-  message    String
-  author_id  Int
-  author     staff               @relation(fields: [author_id], references: [id])
-  created_at DateTime            @default(now())
-  residents  notification_user[]
+  id        Int                 @id @default(autoincrement())
+  message   String
+  authorId  Int                 @map("author_id")
+  author    staff               @relation(fields: [author_id], references: [id])
```

Comment:
I think author_id should be changed to authorId (where it says the fields: [author_id]) as its referring to the camelcase authorId.